examples/multi_gpu/multigpu_papers100m_gcn.py

```diff
@@ -14,15 +14,15 @@
 from torch_geometric.nn import GCNConv
```

Comment:
(nit: Feel free to ignore. I'd name this file `singlenode_multigpu_papers100m_gcn.py`)

Reply:
agreed